### ISSUE TITLE
According to your license""

### ISSUE THREAD

**Issue by Erik-E-Christensen** [D-Multi/Chomper-Basket#1]
Acording to your license I can take your game a distribute it as my own, I would recommend changing that

**Comment by dguisti** [comment]
Yes, but you must give me credit for it if you distribute it! Your right though. I will work on changing that, although I don't think it matters much right now.

**Comment by Erik-E-Christensen** [comment]
Just letting you know!

**Comment by dguisti** [comment]
I'll leave this issue open to remind me to fix the license later when I have time. Thank you for the reccomendation!
